Pipfile

```
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
```

We run Python 3.8 for all the Thoth components:

```suggestion
python_version = "3.8"
```